plugins/polio/api.py

```diff
@@ -246,31 +246,50 @@ def get_year(current_date):
             return today.year
 
     @staticmethod
-    def get_calendar_data(self, year, params):
+    def get_calendar_data(self: Any, year: int, params: Any) -> Any:
+        """
+        Returns filtered rounds from database
+
+            parameters:
+                self (CampaignViewSet): a self CampaignViewSet
+                year (int): a year int
+                params(dictionary): a params dictionary
+            returns:
+                rounds (array of dictionary): a rounds of array of dictionaries
+        """
         countries = params.get("countries") if params.get("countries") is not None else None
         campaign_groups = params.get("campaignGroups") if params.get("campaignGroups") is not None else None
         campaign_type = params.get("campaignType") if params.get("campaignType") is not None else None
         order_by = params.get("order") if params.get("order") is not None else None
         search = params.get("search")
         rounds = Round.objects.filter(started_at__year=year)
+        # Test campaigns should not appear in the xlsx calendar
+        rounds = rounds.filter(campaign__is_test=False)
         if countries:
             rounds = rounds.filter(campaign__country_id__in=countries.split(","))
         if campaign_groups:
             rounds = rounds.filter(campaign__group_id__in=campaign_groups.split(","))
         if campaign_type == "preventive":
             rounds = rounds.filter(campaign__is_preventive=True)
-        if campaign_type == "test":
-            rounds = rounds.filter(campaign__is_test=True)
         if campaign_type == "regular":
             rounds = rounds.filter(campaign__is_preventive=False).filter(campaign__is_test=False)
         if search:
             rounds = rounds.filter(Q(campaign__obr_name__icontains=search) | Q(campaign__epid__icontains=search))
-
+        print(rounds)
```

Comment:
Forgotten?

Reply:
fixed in : https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R277